tests/test_pytest_mpl.py

```diff
@@ -486,3 +490,16 @@ def test_results_always(tmpdir):
                 assert image and not image_exists
                 assert image not in html
                 assert json_res[json_image_key] is None
+
+
+def test_phash(tmpdir):
+    test_file = tmpdir.join("test.py").strpath
+    with open(test_file, "w") as fo:
+        fo.write(TEST_GENERATE)
+
+    command = ["--mpl",
+               "--mpl-kernel=phash",
+               f"--mpl-hash-library={phash_library}",
+               test_file]
+    code = call_pytest(command)
+    assert code == 0
```

Comment:
@Cadair @ConorMacBride @dopplershift 

This perceptual hash test is passing across all versions of python, mpl and platforms.